libcudacxx/include/cuda/__driver/driver_api.h

```diff
@@ -24,7 +24,7 @@
 #define _CCCLRT_GET_DRIVER_FUNCTION_VERSIONED(function_name, versioned_fn_name, major, minor) \
   reinterpret_cast<decltype(versioned_fn_name)*>(__get_driver_entry_point(#function_name, major, minor))
 
-namespace cuda::experimental::__driver
+namespace cuda::__driver
```

Comment:
Question: Do we want to ABI version this API?

If so then we need to put that properly into `libcudacxx\include\cuda\std\__internal\namespaces.h`

We also would want to use a macro like `_CUDA_DRIVER` for the qualification